src/Twig/ImageExtension.php

```diff
@@ -44,6 +46,103 @@ public function getFunctions()
         ];
     }
 
+    /**
+     * {@inheritDoc}
+     */
+    public function getFilters()
+    {
+        return [
+            new TwigFilter('inline_images', [$this, 'inlineImagesInHtmlBlock']),
+        ];
+    }
+
+    public function inlineImagesInHtmlBlock(TwigMarkup $block): TwigMarkup
+    {
+        $hasMatches = preg_match_all('/<img(?:\s+(?P<attrs>.*?))?\\/?>/i', $block, $imageMatches, PREG_SET_ORDER | PREG_OFFSET_CAPTURE);
+        if (false === $hasMatches || 0 === $hasMatches) {
+            return $block;
+        }
+
+        $defaultOptions = [
+            'src' => '',
+            'width' => '',
+            'height' => '',
+            'viewbox-x' => '',
+            'viewbox-y' => '',
+            'class' => [],
+            'attributes' => [
+                'aria-hidden' => 'true',
+                'role' => 'img',
+            ],
+            'title' => '',
+            'directory' => null,
```

Comment:
(flavor) Zou dit sorteren om later minder kans op merge conflicten te krijgen (al verwacht ik die niet in deze bundle, vandaar lage prio)

Reply:
Dit kan niet worden gesorteerd omdat het als `...argument` wordt gebruikt in de aanroep naar de SVG verwerking service. De volgorde moet dus exact hetzelfde zijn als de argument volgorde in de method van die service.
Ik zal hier wel even een comment bij zetten in de code, dan is dat meteen duidelijk.